core/keyserver/keyserver.go

```diff
@@ -89,115 +88,123 @@ func New(tlog tpb.TrillianLogClient,
 // this user and that it is the same one being provided to everyone else.
 // GetUser also supports querying past values by setting the revision field.
 func (s *Server) GetUser(ctx context.Context, in *pb.GetUserRequest) (*pb.GetUserResponse, error) {
```

Comment:
GetUser changes seem out of place in this PR, did this get mixed up with #1131?

Reply:
Yes, it did.  Just rebased to remove it.